frontend/src/components/attendingPage/CardGridAttending.jsx

```diff
@@ -0,0 +1,90 @@
+import Card from "../Card";
+
+
+
+const CardGridAttending = () => {
```

Comment:
Is this for events the user has attended or will be attending. Either way, I would put the keyword Event in, i.e. EventToAttendCard

Reply:
Done